fw-child/default_config.php

```diff
@@ -21,7 +21,7 @@ function custom_global_vars()
     $vars['feedback_email'] = "nullbox@climatedata.ca";
     $vars['training_email'] = "nullbox@climatedata.ca";
     $vars['support_email'] = "nullbox@climatedata.ca";
-    $vars['apps_email'] = "nullbox@climatedata.ca";
+    $vars['apps_email'] = "ccsc-cccs@ec.gc.ca";
```

Comment:
In this file, this email can stay `nullbox@climatedata.ca`. The actual email is to be set in the `local_config.php` file (each server has its own version of the file).

I just did the change in the staging `local_config.php` to use this email.

```suggestion
    $vars['apps_email'] = "nullbox@climatedata.ca";
```